app/services/green_lanes/determine_category.rb

```diff
@@ -1,9 +1,61 @@
 module GreenLanes
   class DetermineCategory
```

Comment:
To help you understand the logic, have a look to the diagram in Jira: https://transformuk.atlassian.net/jira/software/projects/GL/boards/117?assignee=6125108d7a1bfb0071787b51&selectedIssue=GL-580